pkg/collector/corechecks/cluster/kubernetes_openshift.go

```diff
@@ -84,7 +84,8 @@ func computeQuotaRemaining(used, limit v1.ResourceList) v1.ResourceList {
 	remaining := make(map[v1.ResourceName]*resource.Quantity)
 
 	for res, qty := range limit {
-		remaining[res] = qty.Copy()
+		newQuantity := qty.DeepCopy()
```

Comment:
Is this a change due to the dependency having being updated?

Reply:
yes